DJHelper/DJHelper/Model/Event Model/Event Controller/EventController.swift

```diff
@@ -9,18 +9,18 @@
 import Foundation
 import CoreData
 
-class EventController {
-    enum EventErrors: Error {
-        case authorizationError(Error)
-        case noDataError
-        case encodeError(Error)
-        case decodeError(Error)
-        case errorUpdatingEventOnServer(Error)
-        case otherError(Error)
-        case noEventsInServerOrCoreData
-        case couldNotInitializeAnEvent
-    }
+enum EventErrors: Error {
```

Comment:
Interesting. Honest question, can you talk me through moving the enum to outside of the class?